eng/pipelines/templates/stages/archetype-ios-release.yml

```diff
@@ -51,13 +51,17 @@ stages:
               - template: ../variables/globals.yml
             pool:
               name: Azure Pipelines
-              vmImage: $(OSVmImage)
+              vmImage: macOS-10.15
             strategy:
               runOnce:
                 deploy:
                   steps:
                     - script: |
-                        echo "Pretend we are publishing to CocoaPods trunk.""
+                        pod trunk me
```

Comment:
I'm not familiar with the pod tool but is it generally available on mac machines by default?

Reply:
No, you need to install it with a ruby command.

Reply:
It appears to be there by default.